src/main/java/wooteco/subway/domain/DiscountRole.java

```diff
@@ -0,0 +1,45 @@
+package wooteco.subway.domain;
+
+import java.util.Arrays;
+import java.util.function.Predicate;
+import wooteco.subway.domain.strategy.discount.AdolescentDiscountStrategy;
+import wooteco.subway.domain.strategy.discount.DiscountStrategy;
+import wooteco.subway.domain.strategy.discount.FreeDiscountStrategy;
+import wooteco.subway.domain.strategy.discount.KidDiscountStrategy;
+import wooteco.subway.domain.strategy.discount.NonDiscountStrategy;
+
+public enum DiscountRole {
+
+    BABY(age -> age < Constants.KID_STANDARD, new FreeDiscountStrategy()),
+    KID(age -> age >= Constants.KID_STANDARD && age < Constants.ADOLESCENT_STANDARD,
```

Comment:
```suggestion
    KID(age -> Constants.KID_STANDARD <= age && age < Constants.ADOLESCENT_STANDARD,
```
이렇게 표현하면 `Constants.KID_STANDARD <= age < Constants.ADOLESCENT_STANDARD` 와 유사하게 읽혀서 보다 읽기 좋을거 같아요~

Reply:
이 부분 자체가 가독서이 떨어지는 것 같아서 고민이었습니다. 철시가 말씀해주신 내용대로 반영하니 한결 낫네요 😄